src/WikidataReferencer/WikidataReferencerCommand.php

```diff
@@ -172,6 +173,15 @@ public function initServices() {
 						'P58' => 'author',
 						// Organization
 						'P272' => array( 'creator', 'productionCompany' ),
+						// Content
+						'P364' => 'inLanguage',
+						'P674' => 'character',
+						'P840' => 'contentLocation',
+						//Metadata
+						'P166' => 'award',
+						'P1657' => 'contentRating',
+						'P2047' => 'duration',
```

Comment:
Looking at this again duration probably doesn't belong in the ThingReferencer. It will need to have some sort of 
QuantityReferencer created. Maybe the string referencer will do.
The Thing references only created references for wikidata property that have a type of 'item' for example https://www.wikidata.org/wiki/Property:P364